build.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+
+python src/etc/build_rust.py $@
```

Comment:
This will positively not work on any systems where `python == python3 == python3.*`.